Procfile

```diff
@@ -1 +1,2 @@
 web: gunicorn shielddash.wsgi --log-file -
+api: node_modules/.bin/json-server --watch db.json --port 3009
```

Comment:
My understanding is that we only need this for populating the dashboard with sample data. Is that right @spasovski? Do we want to keep this for the time being or take it out? I'm fine either way.